AIMSICD/src/main/res/values/translatable_strings.xml

```diff
@@ -318,7 +318,7 @@
 
 
     <!-- Section Main -->
-    <string name="help_main_neighboring_cells">Shows neighboring cells.</string>
+    <string name="help_main_neighboring_cells">Shows neighbouring cells.</string>
```

Comment:
My Android doesn't agree with you 
![how-to-change-the-language-on-android-from-chinese-to-english-03](https://cloud.githubusercontent.com/assets/2544251/14784368/31d3f8a4-0aed-11e6-83d9-a9b69db4f21d.jpg)


Why are there different English and German dialects supported by Android?